it_config/allowed_external_paths.txt

```diff
@@ -1 +1 @@
-file:/tmp/boxc_test_storage/
\ No newline at end of file
+file:/tmp/boxc_test_storage/
```

Comment:
I'm not really sure what changed here?

Reply:
this was that newline needed for docker :D